client/components/LearnerViewDirective.js

```diff
@@ -164,6 +235,12 @@ tie.directive('learnerView', [function() {
             $scope.previousInstructions.push($scope.instructions);
             $scope.instructions = prompts[currentPromptIndex].getInstructions();
             $scope.nextButtonIsShown = false;
+            // TODO (johnmunoz): DOM not yet updated; is there a better way
+            // than using a timeout?
```

Comment:
I suspect there is indeed a better way to do this, but I need to look into it. Just letting you know I haven't skipped over this!